packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -31,13 +32,15 @@ export const Breadcrumbs = <T extends ElementType = 'nav'>(props: SpiritBreadcru
           items?.map((item, index) => (
             <React.Fragment key={`BreadcrumbsItem_${item.title}`}>
               {index === items.length - 2 && goBackTitle && (
-                <li className="d-tablet-none">
+                <li className="d-tablet-none link-secondary">
```

Comment:
Why are you adding `link-secondary` to `li` element? Link inside `li` is primary.
If `Icon` component inherits `color` from parent (does it?) then the icon should be able to accept `className` prop and you can pass the class name right away to the icon. 

Is it only me who finds it confusing to add `link-secondary` to the `li` element?

Reply:
You are right. I will fix it.

Reply:
I think `text-secondary-default` is what you need. Preferably directly on `.Breadcrumbs` in CSS so we don't to repeat it in HTML.

![obrazek](https://user-images.githubusercontent.com/5614085/189852017-e595f11d-3256-40fe-9a42-29c40647dce4.png)